src/track/track.h

```diff
@@ -267,8 +268,8 @@ class Track : public QObject {
     ///
     /// If the list is empty it tries to complete any pending
     /// import and returns the corresponding status.
-    CueImportStatus importCueInfos(
-            const QList<mixxx::CueInfo>& cueInfos = {});
+    CueImportStatus importCueInfosFromImporter(
```

Comment:
The suffix "FromImporter" is awkward to read and doesn't provide any value. Just importCueInfos().